graphite/graphite.go

```diff
@@ -35,6 +35,9 @@ func (r *Request) CacheKey() string {
 	return fmt.Sprintf("graphite-%d-%d-%s", r.Start.Unix(), r.End.Unix(), targets)
 }
 
+// Query performs a request to Graphite at the given host. host specifies a
+// hostname, and may optionally begin with a scheme (http, https) to specify
+// the protocol (http is the default). header is the headers to send.
```

Comment:
host may also be host:port
see http://golang.org/pkg/net/url/#URL
(i've used this and it works)